crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -339,4 +339,12 @@ impl<'a> TrackedRenderPass<'a> {
         trace!("set blend constant: {:?}", color);
         self.pass.set_blend_constant(wgpu::Color::from(color))
     }
+
+    /// Get the underlying [`RenderPass`].
+    ///
+    /// Resets all tracked state in the process.
+    pub fn inner(&mut self) -> &mut RenderPass<'a> {
+        self.state = Default::default();
```

Comment:
How would this be used? It seems that this could be used to circumvent the checks that `TrackedRenderPass` is designed to prevent. Maybe it would be better to extend `TrackedRenderPass`.

Reply:
If I understand the rest of the code, the checks are specifically optimizations to avoid redundant calls on the underlying RenderPass. By clearing the tracked state, we prevent those checks from firing when they shouldn't and thereby force subsequent calls not to be deduplicated.

Reply:
If so, I think it’s worth adding a comment that `TrackedRenderPass` is an optimization over `RenderPass`, and resetting the state shouldn’t break anything.